--- build.sh	2024-06-18 13:42:24.000000000 +0200
+++ build.sh.new	2024-07-01 16:56:24.473977153 +0200
@@ -523,10 +523,10 @@
     if test -n "$private"; then
       conf_rpath="$idir/lib/build2"
     else
-      conf_rpath="$idir/lib"
+      conf_rpath="$idir/lib64"
     fi
 
-    conf_rpath_stage="$idir/lib"
+    conf_rpath_stage="/usr/lib64"
 
     if test -n "$sudo"; then
       conf_sudo="$sudo"
@@ -607,6 +607,7 @@
 if test "$local" = true; then
 
   run build2/build2/b-boot $verbose configure \
+config.install.lib="/usr/lib64" \
 config.config.hermetic=true \
 config.cxx="$cxx" \
 config.cc.coptions="${compile_ops# }" \
