From b27a854e85ceb2461015c1c7e9aa523ff3f427f4 Mon Sep 17 00:00:00 2001
From: ninetailedtori <ninetailedtori@uwu.gal>
Date: Thu, 9 Oct 2025 15:21:23 +0100
Subject: [PATCH 1/2] bump: fmt 11.0.2 -> 12.0.0

---
 subprojects/fmt.wrap | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/subprojects/fmt.wrap b/subprojects/fmt.wrap
index fd508477f..b5262719d 100644
--- a/subprojects/fmt.wrap
+++ b/subprojects/fmt.wrap
@@ -1,13 +1,13 @@
 [wrap-file]
-directory = fmt-11.0.2
-source_url = https://github.com/fmtlib/fmt/archive/11.0.2.tar.gz
-source_filename = fmt-11.0.2.tar.gz
+directory = fmt-12.0.0
+source_url = https://github.com/fmtlib/fmt/archive/12.0.0.tar.gz
+source_filename = fmt-12.0.0.tar.gz
 source_hash = 6cb1e6d37bdcb756dbbe59be438790db409cdb4868c66e888d5df9f13f7c027f
-patch_filename = fmt_11.0.2-1_patch.zip
-patch_url = https://wrapdb.mesonbuild.com/v2/fmt_11.0.2-1/get_patch
+patch_filename = fmt_12.0.0-1_patch.zip
+patch_url = https://wrapdb.mesonbuild.com/v2/fmt_12.0.0-1/get_patch
 patch_hash = 90c9e3b8e8f29713d40ca949f6f93ad115d78d7fb921064112bc6179e6427c5e
-source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/fmt_11.0.2-1/fmt-11.0.2.tar.gz
-wrapdb_version = 11.0.2-1
+source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/fmt_12.0.0-1/fmt-12.0.0.tar.gz
+wrapdb_version = 12.0.0-1
 
 [provide]
 fmt = fmt_dep

From 667070cabe777d4effe8ecf1274104cf00d44e8a Mon Sep 17 00:00:00 2001
From: Toria <ninetailedtori@uwu.gal>
Date: Mon, 13 Oct 2025 00:26:15 +0100
Subject: [PATCH 2/2] Update fmt.wrap

---
 subprojects/fmt.wrap | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/subprojects/fmt.wrap b/subprojects/fmt.wrap
index b5262719d..15a716ace 100644
--- a/subprojects/fmt.wrap
+++ b/subprojects/fmt.wrap
@@ -2,10 +2,10 @@
 directory = fmt-12.0.0
 source_url = https://github.com/fmtlib/fmt/archive/12.0.0.tar.gz
 source_filename = fmt-12.0.0.tar.gz
-source_hash = 6cb1e6d37bdcb756dbbe59be438790db409cdb4868c66e888d5df9f13f7c027f
+source_hash = aa3e8fbb6a0066c03454434add1f1fc23299e85758ceec0d7d2d974431481e40
 patch_filename = fmt_12.0.0-1_patch.zip
 patch_url = https://wrapdb.mesonbuild.com/v2/fmt_12.0.0-1/get_patch
-patch_hash = 90c9e3b8e8f29713d40ca949f6f93ad115d78d7fb921064112bc6179e6427c5e
+patch_hash = 307f288ebf3850abf2f0c50ac1fb07de97df9538d39146d802f3c0d6cada8998
 source_fallback_url = https://github.com/mesonbuild/wrapdb/releases/download/fmt_12.0.0-1/fmt-12.0.0.tar.gz
 wrapdb_version = 12.0.0-1
 
