diff -Naur xtrs-4.9d+20250818_3a2180c/Makefile xtrs-4.9d+20250818_3a2180c.patched/Makefile
--- xtrs-4.9d+20250818_3a2180c/Makefile	2026-01-09 04:17:18.818819775 -0500
+++ xtrs-4.9d+20250818_3a2180c.patched/Makefile	2026-01-09 05:35:15.791379628 -0500
@@ -188,17 +188,17 @@
 
 install-progs: $(PROGS) $(SRCDIR)$(CASSETTE)
 	$(INSTALL) -d -m 755 $(BINDIR)
-	$(INSTALL) -c -m 755 $(PROGS) $(BINDIR)
+	$(INSTALL) -s -c -m 755 $(PROGS) $(BINDIR)
 	$(INSTALL) -c -m 755 $(SRCDIR)$(CASSETTE) $(BINDIR)/cassette
 
 install-docs: docs
 	$(INSTALL) -d -m 755 $(MANDIR)
-	$(INSTALL) -d -m 755 $(MANDIR)/man1
-	$(INSTALL) -c -m 644 $(SRCDIR)xtrs.man $(MANDIR)/man1/xtrs.1
-	$(INSTALL) -c -m 644 $(SRCDIR)cassette.man $(MANDIR)/man1/cassette.1
-	$(INSTALL) -c -m 644 $(SRCDIR)mkdisk.man $(MANDIR)/man1/mkdisk.1
-	$(INSTALL) -c -m 644 $(SRCDIR)cmddump.man $(MANDIR)/man1/cmddump.1
-	$(INSTALL) -c -m 644 $(SRCDIR)hex2cmd.man $(MANDIR)/man1/hex2cmd.1
+	$(INSTALL) -d -m 755 $(MANDIR)/man6
+	$(INSTALL) -c -m 644 $(SRCDIR)xtrs.man $(MANDIR)/man6/xtrs.6
+	$(INSTALL) -c -m 644 $(SRCDIR)cassette.man $(MANDIR)/man6/cassette.6
+	$(INSTALL) -c -m 644 $(SRCDIR)mkdisk.man $(MANDIR)/man6/mkdisk.6
+	$(INSTALL) -c -m 644 $(SRCDIR)cmddump.man $(MANDIR)/man6/cmddump.6
+	$(INSTALL) -c -m 644 $(SRCDIR)hex2cmd.man $(MANDIR)/man6/hex2cmd.6
 	$(INSTALL) -d -m 755 $(DOCDIR)
 	$(INSTALL) -c -m 644 $(PDFMANPAGES) $(DOCDIR)
 	$(INSTALL) -c -m 644 $(SRCDIR)cpmutil.html $(DOCDIR)
@@ -208,7 +208,7 @@
 
 install-gxtrs: $(GXTRS)
 	$(INSTALL) -d -m 755 $(BINDIR)
-	$(INSTALL) -c -m 755 $(GXTRS) $(BINDIR)
+	$(INSTALL) -s -c -m 755 $(GXTRS) $(BINDIR)
 	$(INSTALL) -d -m 755 $(SHAREDIR)
 	$(INSTALL) -c -m 755 xtrs.glade $(SHAREDIR)
 
diff -Naur xtrs-4.9d+20250818_3a2180c/Makefile.local xtrs-4.9d+20250818_3a2180c.patched/Makefile.local
--- xtrs-4.9d+20250818_3a2180c/Makefile.local	2026-01-09 04:17:18.818819775 -0500
+++ xtrs-4.9d+20250818_3a2180c.patched/Makefile.local	2026-01-09 05:23:49.774444189 -0500
@@ -54,10 +54,10 @@
 # a ROM file is present, it takes precedence over the corresponding
 # built-in ROM.
 
-DEFAULT_ROM = -DDEFAULT_ROM1='"$(SHAREDIR)/level2rom.hex"' \
+DEFAULT_ROM = -DDEFAULT_ROM1='"/usr/share/trs80-roms/level2.rom"' \
 	      -DDEFAULT_ROM1X='"$(SHAREDIR)/esfrom.hex"' \
-              -DDEFAULT_ROM3='"$(SHAREDIR)/romimage.m3"' \
-              -DDEFAULT_ROM4P='"$(SHAREDIR)/romimage.m4p"'
+              -DDEFAULT_ROM3='"/usr/share/trs80-roms/model3.rom"' \
+              -DDEFAULT_ROM4P='"/usr/share/trs80-roms/model4p.rom"'
 
 # If you would like to change where xtrs looks for disk?-? files, edit
 # this line.  "." of course means the current working directory.
diff -Naur xtrs-4.9d+20250818_3a2180c/cassette.man xtrs-4.9d+20250818_3a2180c.patched/cassette.man
--- xtrs-4.9d+20250818_3a2180c/cassette.man	2026-01-09 04:17:18.818819775 -0500
+++ xtrs-4.9d+20250818_3a2180c.patched/cassette.man	2026-01-09 05:24:19.508441391 -0500
@@ -73,7 +73,7 @@
 .\" OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 .\" DEALINGS IN THE SOFTWARE.
 .\"
-.TH cassette 1 2008-06-26 xtrs
+.TH cassette 6 2008-06-26 xtrs
 .SH Name
 cassette \- data cassette image manipulator for xtrs TRS-80 emulator
 .SH Synopsis
diff -Naur xtrs-4.9d+20250818_3a2180c/cmddump.man xtrs-4.9d+20250818_3a2180c.patched/cmddump.man
--- xtrs-4.9d+20250818_3a2180c/cmddump.man	2026-01-09 04:17:18.818819775 -0500
+++ xtrs-4.9d+20250818_3a2180c.patched/cmddump.man	2026-01-09 05:24:27.645440625 -0500
@@ -112,7 +112,7 @@
 .\" OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 .\" DEALINGS IN THE SOFTWARE.
 .\"
-.TH cmddump 1 2017-04-04 xtrs
+.TH cmddump 6 2017-04-04 xtrs
 .SH Name
 cmddump \- simulated TRS-80 CMD file loader
 .SH Synopsis
diff -Naur xtrs-4.9d+20250818_3a2180c/hex2cmd.man xtrs-4.9d+20250818_3a2180c.patched/hex2cmd.man
--- xtrs-4.9d+20250818_3a2180c/hex2cmd.man	2026-01-09 04:17:18.820819775 -0500
+++ xtrs-4.9d+20250818_3a2180c.patched/hex2cmd.man	2026-01-09 05:24:36.796439764 -0500
@@ -74,7 +74,7 @@
 .\" OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 .\" DEALINGS IN THE SOFTWARE.
 .\"
-.TH hex2cmd 1 2008-06-26 xtrs
+.TH hex2cmd 6 2008-06-26 xtrs
 .SH Name
 hex2cmd \- convert Intel hex format to TRS-80 CMD format
 .SH Synopsis
diff -Naur xtrs-4.9d+20250818_3a2180c/mkdisk.man xtrs-4.9d+20250818_3a2180c.patched/mkdisk.man
--- xtrs-4.9d+20250818_3a2180c/mkdisk.man	2026-01-09 04:17:18.821819775 -0500
+++ xtrs-4.9d+20250818_3a2180c.patched/mkdisk.man	2026-01-09 05:24:46.566438844 -0500
@@ -153,7 +153,7 @@
 .\" OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 .\" DEALINGS IN THE SOFTWARE.
 .\"
-.TH mkdisk 1 2017-06-18 xtrs
+.TH mkdisk 6 2017-06-18 xtrs
 .SH Name
 mkdisk \- make a blank emulated floppy disk, hard disk, or Exatron Stringy Floppy wafer for xtrs,
 or add/remove an emulated write-protect tab
diff -Naur xtrs-4.9d+20250818_3a2180c/xtrs.man xtrs-4.9d+20250818_3a2180c.patched/xtrs.man
--- xtrs-4.9d+20250818_3a2180c/xtrs.man	2026-01-09 04:17:18.826819774 -0500
+++ xtrs-4.9d+20250818_3a2180c.patched/xtrs.man	2026-01-09 05:24:56.207437937 -0500
@@ -183,7 +183,7 @@
 .\" OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
 .\" DEALINGS IN THE SOFTWARE.
 .\"
-.TH xtrs 1 2025-05-25 xtrs
+.TH xtrs 6 2025-05-25 xtrs
 .SH Name
 xtrs \- TRS-80 Model I/III/4/4P emulator for the X Window System
 .SH Synopsis
