--- a/panel/CMakeLists.txt
+++ b/panel/CMakeLists.txt
@@ -106,7 +106,7 @@
     ${LIBRARIES}
     ${QTX_LIBRARIES}
     KF6::WindowSystem
-    LayerShellQt::Interface
+    LayerShellQtInterface
     ${STATIC_PLUGINS}
     lxqt-panel-backend-common
 )
--- a/panel/lxqtpanel.cpp
+++ b/panel/lxqtpanel.cpp
@@ -58,7 +58,7 @@
 #include "backends/ilxqtabstractwmiface.h"
 
 
-#include <LayerShellQt/Window>
+#include <LayerShellQt6/Window>
 
 // Turn on this to show the time required to load each plugin during startup
 // #define DEBUG_PLUGIN_LOADTIME
