--- src/launch.py.orig	2026-08-27 14:54:17.000000000 +1000
+++ src/launch.py	2026-09-09 16:07:14.835249280 +1000
@@ -83,10 +83,14 @@
 if scale != 1.0:
     os.environ["QT_SCALE_FACTOR"] = str(scale)
 
-from qt_api import QtCore, QtWidgets
+#from qt_api import QtCore, QtWidgets
+#
+#Qt = QtCore.Qt
+#QApplication = QtWidgets.QApplication
 
-Qt = QtCore.Qt
-QApplication = QtWidgets.QApplication
+from openshot_qt import qt_api
+Qt = qt_api.QtCore.Qt
+QApplication = qt_api.QtWidgets.QApplication
 
 try:
     # This must be done before creating QApplication
