diff --git a/CMakeLists.txt b/CMakeLists.txt
index ba755c80..7c56508b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -275,9 +275,9 @@ option(BUILD_SHARED_LIBS "Build shared libraries" OFF)
 # - to use library package (Ubuntu deb package is old, does not support cmake/find_package)
 # find_package(MbedTLS)
 # - try to find necessary files in system ...
-find_library(MBEDTLS_STATIC_LIB libmbedtls.a /usr/lib /usr/local/lib /usr/local/opt /clang64/lib /clang32/lib)
-find_library(MBEDX509_STATIC_LIB libmbedx509.a /usr/lib /usr/local/lib /usr/local/opt /clang64/lib /clang32/lib)
-find_library(MBEDCRYPTO_STATIC_LIB libmbedcrypto.a /usr/lib /usr/local/lib /usr/local/opt /clang64/lib /clang32/lib)
+find_library(MBEDTLS_STATIC_LIB libmbedtls.so /usr/lib64 /usr/lib)
+find_library(MBEDX509_STATIC_LIB libmbedx509.so /usr/lib64 /usr/lib)
+find_library(MBEDCRYPTO_STATIC_LIB libmbedcrypto.so /usr/lib64 /usr/lib)
 find_path(MBEDTLS_INCLUDE_DIR mbedtls/ssl.h /usr/include /usr/local/include /usr/local/lib /usr/local/opt /clang64/include /clang32/include)
 set(CRYPTO_LIBS ${MBEDTLS_STATIC_LIB} ${MBEDX509_STATIC_LIB} ${MBEDCRYPTO_STATIC_LIB})
 # message("MBEDTLS_STATIC_LIB=${MBEDTLS_STATIC_LIB}")
