diff -Naur meh-0.3+20181021_69f653a.orig/src/gif.c meh-0.3+20181021_69f653a/src/gif.c
--- meh-0.3+20181021_69f653a.orig/src/gif.c	2026-01-14 21:32:38.000000000 -0500
+++ meh-0.3+20181021_69f653a/src/gif.c	2026-01-14 22:35:53.995498911 -0500
@@ -101,7 +101,7 @@
 #endif
 
 #if defined(GIFLIB_MAJOR) && defined(GIFLIB_MINOR) && (GIFLIB_MAJOR == 5 && GIFLIB_MINOR >= 1)
-	DGifCloseFile(g->gif,&ret);
+	ret = DGifCloseFile(g->gif,&ret);
 	if(ret != GIF_OK) {
 #if defined(GIFLIB_MAJOR) && GIFLIB_MAJOR >= 5
 		fprintf(stderr, "GIFLIB: %s\n", GifErrorString(ret));
