diff --git a/data/io.github.TransmissionRemoteGtk.appdata.xml.in b/data/io.github.TransmissionRemoteGtk.appdata.xml.in
index 63cebac..9ed1cb1 100644
--- a/data/io.github.TransmissionRemoteGtk.appdata.xml.in
+++ b/data/io.github.TransmissionRemoteGtk.appdata.xml.in
@@ -22,6 +22,19 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="1.7.0" date="2025-12-26">
+      <description>
+        <p>This release removed some user facing functionality that did not work properly. Core functionality remains unchanged. There are also bug fixes for some minor crashes, and protocol compliance issues.</p>
+        <ul>
+          <li>Remove GeoIP feature (non-functional, used deprecated library)</li>
+          <li>Remove local commands feature (untested, had security issues)</li>
+          <li>Revert support for trashing torrents instead of deleting them</li>
+          <li>Fix a minor crash when adding a torrent by URL</li>
+          <li>Fix a minor crash when using the stats page</li>
+          <li>Ensure JSON is properly UTF-8 encoded from transmission-daemon</li>
+        </ul>
+      </description>
+    </release>
     <release version="1.6.0" date="2023-06-18">
       <description>
           <p>This release adds some behind-the-scenes functionality as well as some small, user-facing bug fixes. The most prominent change is a new backend for interfacing with transmission-daemon that should be more robust and have less bugs.</p>
