From e968f7f521a034de9959b6cae91f5181cf828076 Mon Sep 17 00:00:00 2001
From: Thibault Godouet <yo8192@users.noreply.github.com>
Date: Fri, 18 Apr 2025 21:07:25 +0100
Subject: [PATCH] Fixed fcrontab bug when asked to install a file with relative
 path.

---
 fcrontab.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fcrontab.c b/fcrontab.c
index 75bc1eb..042584e 100644
--- a/fcrontab.c
+++ b/fcrontab.c
@@ -1114,7 +1114,7 @@ main(int argc, char **argv)
                 /* this is just the file name, not the path : complete it */
                 size_t path_len =
                     strlen(orig_dir) + 1 + strlen(argv[file_opt]) + 1;
-                alloc_safe(path_len, fcrontab_file_path);
+                fcrontab_file_path = alloc_safe(path_len, "fcrontab_file_path");
                 snprintf(fcrontab_file_path, path_len, "%s/%s", orig_dir,
                          argv[file_opt]);
             }
