diff -Paur Lmod-9.1.1/pkgs/luafilesystem/Makefile Lmod-9.1.1.new/pkgs/luafilesystem/Makefile
--- Lmod-9.1.1/pkgs/luafilesystem/Makefile	2026-02-12 19:36:28.000000000 +0000
+++ Lmod-9.1.1.new/pkgs/luafilesystem/Makefile	2026-02-13 17:21:26.646117727 +0000
@@ -27,7 +27,10 @@
 	$(CC) $(CFLAGS) $(LIB_OPTION) -o $(LIBRARY) $(OBJ) $(LDFLAGS) -lc
 
 install: $(LIB) all
-	install -m $(MODE_R) *.so* $(LIB)
+	install -m $(MODE_R) $(LIBRARY) $(LIB)
+	$(RM) -f $(LIB)/$(SONAMEV) $(LIB)/$(SONAME)
+	ln -s $(LIBRARY) $(LIB)/$(SONAMEV)
+	ln -s $(SONAMEV) $(LIB)/$(SONAME)
 
 $(LIB):
 	mkdir -p $(LIB)
diff -Paur Lmod-9.1.1/pkgs/term/Makefile Lmod-9.1.1.new/pkgs/term/Makefile
--- Lmod-9.1.1/pkgs/term/Makefile	2026-02-12 19:36:28.000000000 +0000
+++ Lmod-9.1.1.new/pkgs/term/Makefile	2026-02-13 17:24:05.368552767 +0000
@@ -29,7 +29,10 @@
 	$(CC) $(CFLAGS) $(LIB_OPTION) -o $(LIBRARY) $(OBJ) $(LDFLAGS) -lc
 
 install: $(SHARE)/term $(LIB)/term all
-	install -m $(MODE_R) *.so* $(LIB)/term
+	install -m $(MODE_R) $(LIBRARY) $(LIB)/term
+	$(RM) -f $(LIB)/term/$(SONAMEV) $(LIB)/term/$(SONAME)
+	ln -s $(LIBRARY) $(LIB)/term/$(SONAMEV)
+	ln -s $(SONAMEV) $(LIB)/term/$(SONAME)
 	install -m $(MODE_R) term/*.lua $(SHARE)/term
 
 $(LIB)/term $(SHARE)/term:
