diff -Naur hxtools-20251011/smm/extract_d3pkg hxtools-20251011.patched/smm/extract_d3pkg
--- hxtools-20251011/smm/extract_d3pkg	2024-09-01 13:25:33.903652922 -0400
+++ hxtools-20251011.patched/smm/extract_d3pkg	2026-01-08 05:15:14.982003968 -0500
@@ -6,7 +6,7 @@
 
 use Getopt::Long;
 use strict;
-do (($0 =~ m{^(.*)/})[0] || ".")."/extract_dfqshared.pm" ||
+do "/usr/share/hxtools/extract_dfqshared.pm" ||
 	die "Could not load extract_dfqshared.pm: $!\n";
 
 my $ex_out_dir      = ".";
diff -Naur hxtools-20251011/smm/extract_dxhog hxtools-20251011.patched/smm/extract_dxhog
--- hxtools-20251011/smm/extract_dxhog	2024-09-01 13:43:27.477147824 -0400
+++ hxtools-20251011.patched/smm/extract_dxhog	2026-01-08 05:15:35.094002075 -0500
@@ -6,7 +6,7 @@
 
 use Getopt::Long;
 use strict;
-do (($0 =~ m{^(.*)/})[0] || ".")."/extract_dfqshared.pm" ||
+do "/usr/share/hxtools/extract_dfqshared.pm" ||
 	die "Could not load extract_dfqshared.pm: $!\n";
 
 my $ex_out_dir      = ".";
diff -Naur hxtools-20251011/smm/extract_qupak hxtools-20251011.patched/smm/extract_qupak
--- hxtools-20251011/smm/extract_qupak	2024-09-01 13:25:34.233652128 -0400
+++ hxtools-20251011.patched/smm/extract_qupak	2026-01-08 05:15:57.639999953 -0500
@@ -6,7 +6,7 @@
 
 use Getopt::Long;
 use strict;
-do (($0 =~ m{^(.*)/})[0] || ".")."/extract_dfqshared.pm" ||
+do "/usr/share/hxtools/extract_dfqshared.pm" ||
 	die "Could not load extract_dfqshared.pm: $!\n";
 
 my $ex_out_dir      = ".";
