From 3f107fcf723d7ed1a075329bd571fa0a7396f5e2 Mon Sep 17 00:00:00 2001
From: Hiltjo Posthuma <hiltjo@codemadness.org>
Date: Mon, 9 Jan 2023 18:27:12 +0100
Subject: [PATCH 6/8] Makefile: be verbose

---
 Makefile | 40 ++++++++++++++++------------------------
 1 file changed, 16 insertions(+), 24 deletions(-)

diff --git a/Makefile b/Makefile
index 56e6367..731f71b 100644
--- a/Makefile
+++ b/Makefile
@@ -18,43 +18,35 @@ config.h:
 	cp config.def.h config.h
 
 .c.o:
-	@echo CC $<
-	@${CC} -c ${CFLAGS} $<
+	${CC} -c ${CFLAGS} $<
 
 ${OBJ}: config.h config.mk
 
 sent: ${OBJ}
-	@echo CC -o $@
-	@${CC} -o $@ ${OBJ} ${LDFLAGS}
+	${CC} -o $@ ${OBJ} ${LDFLAGS}
 
 cscope: ${SRC} config.h
-	@echo cScope
-	@cscope -R -b || echo cScope not installed
+	cscope -R -b || echo cScope not installed
 
 clean:
-	@echo cleaning
-	@rm -f sent ${OBJ} sent-${VERSION}.tar.gz
+	rm -f sent ${OBJ} sent-${VERSION}.tar.gz
 
 dist: clean
-	@echo creating dist tarball
-	@mkdir -p sent-${VERSION}
-	@cp -R LICENSE Makefile config.mk config.def.h ${SRC} sent-${VERSION}
-	@tar -cf sent-${VERSION}.tar sent-${VERSION}
-	@gzip sent-${VERSION}.tar
-	@rm -rf sent-${VERSION}
+	mkdir -p sent-${VERSION}
+	cp -R LICENSE Makefile config.mk config.def.h ${SRC} sent-${VERSION}
+	tar -cf sent-${VERSION}.tar sent-${VERSION}
+	gzip sent-${VERSION}.tar
+	rm -rf sent-${VERSION}
 
 install: all
-	@echo installing executable file to ${DESTDIR}${PREFIX}/bin
-	@mkdir -p ${DESTDIR}${PREFIX}/bin
-	@cp -f sent ${DESTDIR}${PREFIX}/bin
-	@chmod 755 ${DESTDIR}${PREFIX}/bin/sent
-	@echo installing manual page to ${DESTDIR}${MANPREFIX}/man1
-	@mkdir -p ${DESTDIR}${MANPREFIX}/man1
-	@cp sent.1 ${DESTDIR}${MANPREFIX}/man1/sent.1
-	@chmod 644 ${DESTDIR}${MANPREFIX}/man1/sent.1
+	mkdir -p ${DESTDIR}${PREFIX}/bin
+	cp -f sent ${DESTDIR}${PREFIX}/bin
+	chmod 755 ${DESTDIR}${PREFIX}/bin/sent
+	mkdir -p ${DESTDIR}${MANPREFIX}/man1
+	cp sent.1 ${DESTDIR}${MANPREFIX}/man1/sent.1
+	chmod 644 ${DESTDIR}${MANPREFIX}/man1/sent.1
 
 uninstall:
-	@echo removing executable file from ${DESTDIR}${PREFIX}/bin
-	@rm -f ${DESTDIR}${PREFIX}/bin/sent
+	rm -f ${DESTDIR}${PREFIX}/bin/sent
 
 .PHONY: all options clean dist install uninstall cscope
-- 
2.54.0

