--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -67,7 +67,7 @@
 add_subdirectory(src)
 add_subdirectory(tests)
 
-set(CMAKECONFIG_INSTALL_DIR ${KDE_INSTALL_CMAKEPACKAGEDIR}/LayerShellQt)
+set(CMAKECONFIG_INSTALL_DIR ${KDE_INSTALL_CMAKEPACKAGEDIR}/LayerShellQt6)
 install(EXPORT LayerShellQtTargets
         NAMESPACE LayerShellQt::
         DESTINATION ${CMAKECONFIG_INSTALL_DIR}
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -76,7 +76,7 @@
 install(FILES
     ${LayerShellQt_HEADERS}
     ${CMAKE_CURRENT_BINARY_DIR}/LayerShellQt/layershellqt_export.h
-    DESTINATION ${KDE_INSTALL_INCLUDEDIR}/LayerShellQt COMPONENT Devel
+    DESTINATION ${KDE_INSTALL_INCLUDEDIR}/LayerShellQt6 COMPONENT Devel
 )
 
 add_subdirectory(declarative)
